tests/index.html

```diff
@@ -24,24 +25,23 @@
         along with Kiwix (file LICENSE-GPLv3.txt).  If not, see <http://www.gnu.org/licenses/>
         -->
 
-        <link rel="stylesheet" href="tests/qunit-2.3.2.css" />
-        <script src="tests/qunit-2.3.2.js"></script>
+        <link rel="stylesheet" href="node_modules/qunit/qunit/qunit.css" />
+        <script src="node_modules/qunit/qunit/qunit.js"></script>
+        <script>
+            QUnit.config.autostart = false;
+        </script>
 
         <!-- Using require.js, a module system for javascript, include the
              js files. This loads "main.js", which in turn can load other
              files, all handled by require.js:
              http://requirejs.org/docs/api.html#jsfiles -->
-        <script type="text/javascript" src="www/js/lib/xzdec.js"></script>
-        <script type="text/javascript"
-                data-main="tests/init.js"
-        src="www/js/lib/require.js"></script>
-
-
+        <script src="www/js/lib/xzdec.js"></script>
```

Comment:
@Krinkle You can remove this script (xzdec.js) as it is now implicitly loaded by the RequireJS configuration for the tests. We have removed this from the equivalent `index.html` for the app. I've tested this on master, and QUnit tests run fine without it (and clearly those tests load xzdec wrapper and then xzdec). I realize this isn't part of this PR, but might as well be done while we're modifying the file.

Reply:
OK.

Reply:
To confirm, are you saying `xzdec` is no longer needed in general, or just no longer needed to be loaded from here at this line? I think it is still used, it is just loaded automatically now from `define/zimArchive > zimfile > xzdec_wrapper > xzdec`.

Reply:
@Krinkle You understand correctly: it *is* used, and is loaded automatically from the define statement when needed. So doesn't need to be loaded the old-fashioned way any more (I should have got rid of this script element at the time I wrote the zstddec_wrapper, but forgot).